migration/esr128.md

```diff
@@ -0,0 +1,575 @@
+  * 「一般」→「言語と外観」→「言語」→「言語設定」→「上へ」
+    * `pref.browser.language.disable_button.up`
+  * 「一般」→「言語と外観」→「言語」→「言語設定」→「下へ」
+    * `pref.browser.language.disable_button.down`
+  * 「一般」→「言語と外観」→「言語」→「言語設定」→「削除」
+    * `pref.browser.language.disable_button.remove`
+  * 「一般」→「ファイルとプログラム」→「プログラム」→ファイルの種類と取り扱い方法の一覧
+    * `pref.downloads.disable_button.edit_actions`
+  * 「一般」→「Firefoxの更新」→「更新履歴を表示」
+    * `app.update.disable_button.showUpdateHistory`
+  * 「一般」→「ネットワーク設定」→「接続設定」→「自動プロキシー設定スクリプト」→「再読み込み」
+    * `pref.advanced.proxies.disable_button.reload`
+  * 「ホーム」→「新しいウィンドウとタブ」→「カスタムURL」選択時の「現在のタブグループを使う」
+    * `pref.browser.homepage.disable_button.current_page`
+  * 「ホーム」→「新しいウィンドウとタブ」→「カスタムURL」選択時の「ブックマークを使う」
+    * `pref.browser.homepage.disable_button.bookmark_page`
+  * 「ホーム」→「既定値に戻す」
+    * `pref.browser.homepage.disable_button.restore_default`
+  * 「プライバシーとセキュリティ」→「ブラウザープライバシー」→「強化型トラッキング防止機能」→「例外を管理」
+    * `pref.privacy.disable_button.tracking_protection_exceptions`
+  * 「プライバシーとセキュリティ」→「ブラウザープライバシー」→「Cookieとサイトデータ」→「例外を管理」
+    * `pref.privacy.disable_button.cookie_exceptions`
+  * 「プライバシーとセキュリティ」→「ブラウザープライバシー」→「パスワード」→「例外サイト」
+    * `pref.privacy.disable_button.view_passwords_exceptions`
+  * 「プライバシーとセキュリティ」→「ブラウザープライバシー」→「パスワード」→「保存されたパスワード」
+    * `pref.privacy.disable_button.view_passwords`
+  * 「プライバシーとセキュリティ」→「セキュリティ」→「証明書」→「証明書を表示」
+    * `security.disable_button.openCertManager` （Firefox 121）
+  * 「プライバシーとセキュリティ」→「セキュリティ」→「証明書」→「セキュリティデバイス」
+    * `security.disable_button.openDeviceManager` （Firefox 121）
+* デスクトップ通知について、以下の挙動を `Preferences` ポリシーで制御可能になった。（Firefox 122）
+  * Webサイトのアイコン（favicon）を表示するかどうか `alerts.showFavicons`（初期値：`false`）
+  * システムの通知機能で表示するかどうか `alerts.useSystemBackend`（初期値：`true`）
+  * システムの通知機能で、一度閉じた通知を通知一覧から再表示できるようにするかどうか `alerts.useSystemBackend.windows.notificationserver.enabled`（初期値：`true`）
+* アプリケーションメニューの「動作しないサイトを報告」からWeb互換性の報告を行えるようになった。（Firefox 123）  
+  ![（画像：アプリケーションメニューからの「動作しないサイトを報告」時のUIのスクリーンショット。）](./esr128/report-from-appmenu.png)
+  * この機能は、`DisableFeedbackCommands` ポリシーを用いて無効化できます。
+* いずれの項目も選択されていないラジオボタンのグループにおいて、Tabキーでのフォーカス移動時には最初の項目にのみフォーカスが当たるようになった（項目を選択する場合は上下カーソルキーを使用）。（Firefox 125）
+* テレメトリー（使用状況の統計情報の収集）において、検索の使用状況を個人と紐付けない形で「スポーツ」「ビジネス」「旅行」などの20のカテゴリー単位で記録・送信するようになった。（Firefox 126）（[詳細情報](https://blog.mozilla.org/en/products/firefox/firefox-search-update/)）
+  * この挙動は、`DisableTelemetry` ポリシーを `true` に設定することで、他の統計情報の収集・送信と併せて無効化できます。
+* 設定画面の「一般」→「起動」→「コンピューターの起動時にFirefoxを自動的に開く」で、Firefoxを現在のユーザーのスタートアップに登録できるようになった。（Firefox 127）（[詳細情報](https://support.mozilla.org/kb/open-firefox-automatically-when-you-start-computer)）
+  * 登録先は  
+    `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run`  
+    です。
+  * この機能は、Windows上で、  
+    `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion`  
+    `\Explorer\StartupApproved\Run`  
+    でアプリケーションの自動起動が許可されている場合にのみ使用できます。
+* タブのコンテキストメニュー、およびタブバー端の全タブの一覧を表示するドロップダウンメニューに「重複タブを閉じる」コマンドが追加された。（Firefox 127）
```

Comment:
```suggestion
* タブのコンテキストメニュー、およびタブバー端の全タブの一覧を表示するドロップダウンメニューに「重複タブを閉じる」コマンドが追加された。（Firefox 127）
![（画像：コンテキストメニュー「重複タブを閉じる」のスクリーンショット）](esr128/tab-context-menu-sample.png)
![（画像：ドロップダウンメニュー「重複タブを閉じる」のスクリーンショット）](esr128/tab-dropdown-menu-sample.png)
```
Although that will be your choice, explaining graphically by adding a few screenshots as an example would be better.